packages/api/test/util/describeE2E.ts

```diff
@@ -4,24 +4,35 @@
 
 const WS_ENDPOINTS = {
   local: 'ws://127.0.0.1:9944/',
-  'substrate-master': 'ws://127.0.0.1:9945/',
-  'substrate-1.0': 'ws://127.0.0.1:9946/',
-  'substrate-2.0': 'ws://127.0.0.1:9947/',
-  'polkadot-master': 'ws://127.0.0.1:9948/',
-  'polkadot-alexander': 'ws://127.0.0.1:9949/',
+  'docker-substrate-master': 'ws://127.0.0.1:9945/',
+  'docker-substrate-1.0': 'ws://127.0.0.1:9946/',
+  'docker-substrate-2.0': 'ws://127.0.0.1:9947/',
+  'docker-polkadot-master': 'ws://127.0.0.1:9948/',
+  'docker-polkadot-alexander': 'ws://127.0.0.1:9949/',
   'remote-polkadot-alexander': 'wss://poc3-rpc.polkadot.io/',
   'remote-substrate-1.0': 'wss://substrate-rpc.parity.io/'
 };
 
 type WsName = keyof typeof WS_ENDPOINTS;
 
 interface Options {
-  except?: WsName[]; // Only one of the two keys `either` `only` should be set
+  except?: WsName[]; // Only one of the two keys `except` `only` should be set
   only?: WsName[];
   apiType?: 'promise' | 'rxjs';
```

Comment:
You can actually remove apiType in all tests, I added it but it's actually useless

Reply:
removed